content/en/docs/reference/glossary/qos-class.md

```diff
@@ -0,0 +1,24 @@
+---
+title: QoSClass
```

Comment:
Never saw 'QoSClass' anywhere before ... shall we use 'QoS Class' instead? 'QoSClass' reads like a hardcoded resource type.